src/Argu.Dotnetcli.Tests/project.json

```diff
@@ -0,0 +1,42 @@
+{
+    "version": "1.0.0-*",
+    "buildOptions": {
+        "debugType": "portable",
+        "emitEntryPoint": true,
+        "compilerName": "fsc",
+        "define": [
+            "CORE_CLR"
+        ],
+        "compile": {
+            "includeFiles": [
+                "Program.fs"
+            ]
+        }
+    },
+    "tools": {
+        "dotnet-compile-fsc": {
+            "version": "1.0.0-preview2-*",
+            "imports": "dnxcore50"
+        }
+    },
+    "frameworks": {
+        "netstandard1.6" : { 
+            "dependencies": {
+                "Argu":  { "version": "3.0.0-beta02", "type": "build" }
+            },
+            "imports": [
+                "portable-net45+win8",
+                "dnxcore50"
+            ]
+        },
+        "net46" : { 
+            "dependencies": {
+                "Argu":  { "version": "3.0.0-beta02", "type": "build" }
```

Comment:
if deps are shared, you can put these outside `frameworks`, like that are shared with all frameworks (and frameworks like `net46` or `netstandard1.6` can add more)